docs/source/variables.ipynb

```diff
@@ -0,0 +1,182 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Variable Types\n",
+    "\n",
+    "A Variable is analogous to a column in table in a relational database. When running DFS, Featuretools will attempt to infer the types of variable present in your data. DFS also allows for specifying the variable types.\n",
```

Comment:
> When running DFS, Featuretools will attempt to infer the types of variable present in your data.

types of _variables_

I think in this context DFS should be replaced with Entity or EntitySet

Reply:
Fixed